src/main/java/org/folio/innreach/domain/entity/AgencyLocationLscMapping.java

```diff
@@ -0,0 +1,52 @@
+package org.folio.innreach.domain.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.folio.innreach.domain.entity.base.Identifiable;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.List;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString(exclude = "centralServerMapping")
+@Entity
+@Table(name = "agency_location_lsc_mapping")
+public class AgencyLocationLscMapping implements Identifiable<UUID> {
+
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  @Id
+  private UUID id;
+
+  private UUID locationId;
+
+  private UUID libraryId;
+
+  private String localServerCode;
+
+  @OneToMany(
+    cascade = CascadeType.ALL,
+    fetch = FetchType.EAGER,
```

Comment:
I believe the fetch type should be "LAZY", what do you think?

Reply:
According to UI implementation, these entities are going to be fetched all together, I.e.
```
{
  "libraryId": "ef261062-e582-43d0-a1fc-c32dfca1da22",
  "locationId": "aa58c309-4522-4b46-8d1e-0396ee493460",
  "localServers": [
    {
      "localCode": "5publ",
      "localServerLibraryId": "ef261062-e582-43d0-a1fc-c32dfca1da22",
      "localServerLocationId": "aa58c309-4522-4b46-8d1e-0396ee493460",
      "agencyCodeMappings": [
        {
          "agencyCode": "5east",
          "libraryId": "70cf3473-77f2-4f5c-92c3-6489e65769e4",
          "locationId": "99b0d4e2-a5ec-46a1-a5ea-1080e609f969"
        }
      ]
    }
  ]
}
```
So there is no need to fetch them without child entities

Reply:
Ok, how about an update? If you need to update basic fields like localCode, libraryId or locationId OR add a new mapping to the agencyCodeMappings collection, you don't need to fetch the whole entity with the agencyCodeMappings, right? But you will :) since the fetch type is EAGER.

As far as I know, the best practice is to always use the LAZY fetch type and use JPQL JOIN FETCH queries when you need to get an entity with childs.

Please, correct me if I'm wrong.

Reply:
Switched to use `Set` collections for nested entities instead of `List` as there are limitations joining multiple lists of entities in hibernate